diff --git a/extern/goocanvasmm/goocanvasmm/goocanvas/goocanvasmm/canvas.cc b/extern/goocanvasmm/goocanvasmm/goocanvas/goocanvasmm/canvas.cc
index 3fa2925..30c0b3b 100644
--- a/extern/goocanvasmm/goocanvasmm/goocanvas/goocanvasmm/canvas.cc
+++ b/extern/goocanvasmm/goocanvasmm/goocanvas/goocanvasmm/canvas.cc
@@ -312,7 +312,7 @@ void Canvas_Class::item_created_callback(GooCanvas* self, GooCanvasItem* p0, Goo
 
 Glib::ObjectBase* Canvas_Class::wrap_new(GObject* o)
 {
-  return manage(new Canvas((GooCanvas*)(o)));
+  return Gtk::manage(new Canvas((GooCanvas*)(o)));
 
 }
 
diff --git a/extern/goocanvasmm/goocanvasmm/goocanvas/goocanvasmm/points.cc b/extern/goocanvasmm/goocanvasmm/goocanvas/goocanvasmm/points.cc
index 11a0c00..e3236fe 100644
--- a/extern/goocanvasmm/goocanvasmm/goocanvas/goocanvasmm/points.cc
+++ b/extern/goocanvasmm/goocanvasmm/goocanvas/goocanvasmm/points.cc
@@ -55,7 +55,7 @@ Points::get_num_points() const
 }
 
 void
-Points::Points::set_coordinate( int index, double x, double y )
+Points::set_coordinate( int index, double x, double y )
 {
   gobj()->coords[2*index] = x ;
   gobj()->coords[2*index+1] = y ;
