From 44226b50c361dec8927557bec8105d504bf62ef2 Mon Sep 17 00:00:00 2001
From: "Daniele E. Domenichelli" <daniele.domenichelli@iit.it>
Date: Wed, 23 Jun 2021 12:38:22 +0200
Subject: [PATCH 1/2] extern/websocket: Add missing header

---
 extern/websocket/WebSocket/WebSocket/WebSocket.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/extern/websocket/WebSocket/WebSocket/WebSocket.cpp b/extern/websocket/WebSocket/WebSocket/WebSocket.cpp
index 0420104f3..dcb6244cd 100644
--- a/extern/websocket/WebSocket/WebSocket/WebSocket.cpp
+++ b/extern/websocket/WebSocket/WebSocket/WebSocket.cpp
@@ -13,6 +13,7 @@
 #include <iostream>
 #include <string>
 #include <vector>
+#include <cstring>
 
 using namespace std;
 
-- 
2.30.2

