From 85ca0d444c8462cd0d9da9130f82d6cfbf02f7fa Mon Sep 17 00:00:00 2001
From: "Daniele E. Domenichelli" <daniele.domenichelli@iit.it>
Date: Wed, 13 Nov 2013 11:51:20 +0100
Subject: [PATCH 12/22] Documentation tweaks
---
 extern/xmlrpcpp/xmlrpcpp/XmlRpcUtil.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/extern/xmlrpcpp/xmlrpcpp/src/XmlRpcUtil.h b/extern/xmlrpcpp/xmlrpcpp/src/XmlRpcUtil.h
index c44f553..e48e6a0 100644
--- a/extern/xmlrpcpp/xmlrpcpp/src/XmlRpcUtil.h
+++ b/extern/xmlrpcpp/xmlrpcpp/src/XmlRpcUtil.h
@@ -30,7 +30,7 @@ namespace XmlRpc {
   class XmlRpcUtil {
   public:
     // hokey xml parsing
-    //! Returns contents between <tag> and </tag>, updates offset to char after </tag>
+    //! Returns contents between \<tag\> and \</tag\>, updates offset to char after \</tag\>
     static std::string parseTag(const char* tag, std::string const& xml, int* offset);
 
     //! Returns true if the tag is found and updates offset to the char after the tag
-- 
2.9.3

