From 53ea746a7fa2d2603df46763b8cb380dcc2407ab Mon Sep 17 00:00:00 2001
From: "Daniele E. Domenichelli" <daniele.domenichelli@iit.it>
Date: Thu, 27 Aug 2015 16:41:10 +0200
Subject: [PATCH 14/22] [Cleanup]

---
 extern/xmlrpcpp/xmlrpcpp/XmlRpcValue.h | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/extern/xmlrpcpp/xmlrpcpp/src/XmlRpcValue.h b/extern/xmlrpcpp/xmlrpcpp/src/XmlRpcValue.h
index fae12c2..2d4bda8 100644
--- a/extern/xmlrpcpp/xmlrpcpp/src/XmlRpcValue.h
+++ b/extern/xmlrpcpp/xmlrpcpp/src/XmlRpcValue.h
@@ -1,12 +1,14 @@
-
-#ifndef _XMLRPCVALUE_H_
-#define _XMLRPCVALUE_H_
 //
 // XmlRpc++ Copyright (c) 2002-2003 by Chris Morley
 //
 // Summary for YARP:
 // Copyright: 2002, 2003 Chris Morley
 // CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+#ifndef _XMLRPCVALUE_H_
+#define _XMLRPCVALUE_H_
+
+
 #if defined(_MSC_VER)
 # pragma warning(disable:4786)    // identifier was truncated in debug info
 #endif
@@ -182,7 +184,7 @@ namespace YarpXmlRpc {
       ValueArray*   asArray;
       ValueStruct*  asStruct;
     } _value;
-    
+
   };
 } // namespace YarpXmlRpc
 
-- 
2.9.3

