From 95238ffbcd164594ace75baff278ac936a14484b Mon Sep 17 00:00:00 2001
From: "Daniele E. Domenichelli" <daniele.domenichelli@iit.it>
Date: Tue, 11 Oct 2016 17:48:10 +0200
Subject: [PATCH 17/22] xmlrpc++: Fix tmEq() method

---
 extern/xmlrpcpp/xmlrpcpp/XmlRpcValue.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/extern/xmlrpcpp/xmlrpcpp/src/XmlRpcValue.cpp b/extern/xmlrpcpp/xmlrpcpp/src/XmlRpcValue.cpp
index 7efb039..2e176a6 100644
--- a/extern/xmlrpcpp/xmlrpcpp/src/XmlRpcValue.cpp
+++ b/extern/xmlrpcpp/xmlrpcpp/src/XmlRpcValue.cpp
@@ -148,7 +148,7 @@ namespace YarpXmlRpc {
   // Predicate for tm equality
   static bool tmEq(struct tm const& t1, struct tm const& t2) {
     return t1.tm_sec == t2.tm_sec && t1.tm_min == t2.tm_min &&
-            t1.tm_hour == t2.tm_hour && t1.tm_mday == t1.tm_mday &&
+            t1.tm_hour == t2.tm_hour && t1.tm_mday == t2.tm_mday &&
             t1.tm_mon == t2.tm_mon && t1.tm_year == t2.tm_year;
   }
 
-- 
2.9.3

